src/test/java/com/intive/patronage22/szczecin/retroboard/service/BoardServiceTest.java

```diff
@@ -471,11 +472,50 @@ void assignUsersToBoardShouldReturnFailedEmails() throws JSONException {
         assertEquals(failedEmails.get(0), usersEmails.get(1));
     }
 
-    private Board buildBoard(final User user) {
+    @Test
+    void patchBoardShouldReturnBadRequestWhenBoardStateNotCreated() {
+        // given
+        final var uid = "1234";
+        final var email = "John@test.pl";
+        final var user = new User(uid, email, "john14", Set.of());
+        final var state = EnumStateDto.VOTING;
```

Comment:
you don't need to create state as separate variable if it's used only in line below
```suggestion
        final var board = buildBoard(user, EnumStateDto.CREATED);
```

Reply:
done